### PR TITLE
doc: installation: add section about terminal env

### PR DESCRIPTION
**Pull request by greg-fer**
Added a section about using terminal environment, as a target for different references in docs.
Requested by @carlescufi.

### CODE REVIEW

**Review by shanthanordic — APPROVED** [review]
LGTM

**Review by gmarull — APPROVED** [review]

File: doc/nrf/installation/updating.rst [hunk 1]
```diff
@@ -16,6 +16,52 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _using_toolchain_environment:
+
+Using toolchain environment
```
Comment (carlescufi) [inline doc/nrf/installation/updating.rst:21]: I think this needs the phrasing "command-line interface" somewhere. Both in the title and also in the paragraph in line 24. It is not clear enough here that "environment" really means "cli environment"

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: doc/nrf/installation/updating.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/installation/updating.rst:21]: True, let me edit.

**Comment by greg-fer** [comment]
@carlescufi , can you have another look?

@b-gent , can you add an entry in #15961 in the documentation section? Thanks.
```
* Added a section about :ref:`using_toolchain_environment` on the :ref:`updating` page.
```

**Review by carlescufi — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**